### DISCUSSION TITLE
How to extract the RPN and ROI heads from the "TRAINED" Faster RCNN Model

### DISCUSSION THREAD

**Discussion by eddydw** [facebookresearch/detectron2#4904] [General]
I want to split the RPN and ROI Heads from the rained Faster RCNN model?
How can I do that?

FYI :
The reason for doing that is because I want to deploy the model on an heterogeneous edge device. The graph has to be split such I can deploy the 3 subgraphs on 3 different heterogeneous compute engines on my edge device.

**Comment by vkrishnamurthy11** [comment]
If I understand correctly, you want to be able to run the RPN, save the output of the RPN and then run that saved output on the ROI heads separately?
